### ISSUE TITLE
What does TPOT stand for?

### ISSUE THREAD

**Issue by evanmiller29** [EpistasisLab/tpot#907]
As TPOT is obviously an acronym it would be useful to understand what it stands for.
A quick google shows that TPOT stands for the people of truth - is that the case?

## Possible fix

Include in README on Github repo

**Comment by weixuanfu** [comment]
Thanks for the suggestion. It stand for **T**ree-based **P**ipeline **O**ptimization **T**ool mentioned in the first paper of TPOT. I think this acronym info should be added into README on Github repo.

```
@inproceedings{OlsonGECCO2016,
    author = {Olson, Randal S. and Bartley, Nathan and Urbanowicz, Ryan J. and Moore, Jason H.},
    title = {Evaluation of a Tree-based Pipeline Optimization Tool for Automating Data Science},
    booktitle = {Proceedings of the Genetic and Evolutionary Computation Conference 2016},
    series = {GECCO '16},
    year = {2016},
    isbn = {978-1-4503-4206-3},
    location = {Denver, Colorado, USA},
    pages = {485--492},
    numpages = {8},
    url = {http://doi.acm.org/10.1145/2908812.2908918},
    doi = {10.1145/2908812.2908918},
    acmid = {2908918},
    publisher = {ACM},
    address = {New York, NY, USA},
}
```

**Comment by evanmiller29** [comment]
@weixuanfu - happy to do this

**Comment by apachaves** [comment]
@weixuanfu, I've sent a simple PR for this as a test. I want to learn how to contribute and this is my first time doing this, so I didn't want to change much to break anything important. Don't know if you can help me.

**Comment by weixuanfu** [comment]
This was fixed in PR #924
